build-system/babel-plugins/babel-plugin-transform-promise-resolve/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```

Comment:
nit: 2020.

Q: does having the correct current year matter? i.e. is this worth the time spent commenting/fixing?